pkg/controller/statusmanager/status_manager_test.go

```diff
@@ -253,6 +253,59 @@ func TestStatusManager_set(t *testing.T) {
 	}
 }
 
+func TestStatusManager_set_kuryr(t *testing.T) {
+	client := fake.NewFakeClient()
+	status := New(client, "testing", "")
+
+	// make the network.operator object
+	log.Print("Creating Network Operator Config")
```

Comment:
leftover?

Reply:
I've copied this from TestStatusManager_set() above. It proves to be useful when debugging the tests.